### ISSUE TITLE
Sourceror.parse_expression fails if the code only contains comments

### ISSUE THREAD

**Issue by zachdaniel** [doorgan/sourceror#63]
For example: `Sourceror.parse("# hello_world()")`.

I'm not sure what the path to contributing a fix would be, but happy to fix it with some guidance.

**Comment by NickNeck** [comment]
I have fixed a similar issue here https://github.com/doorgan/sourceror/pull/58/files . Maybe that fix needs some additions.

**Comment by NickNeck** [comment]
@zachdaniel I have tested it and for me it looks fine.
```elixir
iex(1)> Sourceror.parse_string!("# hello_world()")
{:__block__,
 [
   __sourceror__: %{trailing_block: true},
   trailing_comments: [
     %{
       column: 1,
       line: 1,
       next_eol_count: 0,
       previous_eol_count: 1,
       text: "# hello_world()"
     }
   ],
   leading_comments: [],
   line: 1
 ], [{:__block__, [trailing_comments: [], leading_comments: []], []}]}
```

**Comment by zachdaniel** [comment]
Yep, looks like its been fixed in github but not in the latest published hex version :)

**Comment by zachdaniel** [comment]
Thanks!

**Comment by zachdaniel** [comment]
Would be good to cut a release at some point if possible, as this can cause some pain points for Ash users that are using our provided formatter plugin, which fails on files that are only comments :)

**Comment by doorgan** [comment]
@zachdaniel I was meaning to cut a patch release after fixing a remaining issue with #53 but then life got in the way :)
I just published v0.11.2 including the fix for this particular issue so you can have a quick fix for Ash. I'll release a new patch when a fix the remaining issue in #53.

**Comment by zachdaniel** [comment]
Thanks so much!
